tests/dashboard/components/file/test_views.py

```diff
@@ -0,0 +1,405 @@
+            "status": "backlog",
+            "size": 0.032919883728027344,
+            "tags": [],
+            "bulk_extractor_reports": [],
+            "format": [
+                {"puid": "x-fmt/111", "format": "Generic TXT", "group": "Text (Plain)"}
+            ],
+            "pending_deletion": False,
+        },
+    )
+    response = admin_client.get(reverse("file:file_details", args=[file.uuid]))
+    assert response.status_code == 200
+
+    content = json.loads(response.content.decode("utf8"))
+    assert content["id"] == str(file.uuid)
+    assert content["title"] == "LICENSE"
+
+
+@pytest.mark.parametrize(
+    "mock_file_info, status_code",
+    [
+        ("get_transfer_file_info returned no exact results", 404),
+        ("unknown error", 500),
+    ],
+    ids=["no exact results", "unknown error"],
+)
+def test_file_details_handles_exceptions(
+    admin_client, dashboard_uuid, file, mocker, mock_file_info, status_code
+):
+    mocker.patch("elasticSearchFunctions.get_client")
+    mocker.patch(
+        "elasticSearchFunctions.get_transfer_file_info",
+        side_effect=elasticSearchFunctions.ElasticsearchError(mock_file_info),
+    )
+
+    response = admin_client.get(reverse("file:file_details", args=[file.uuid]))
+    assert response.status_code == status_code
+
+    assert json.loads(response.content.decode("utf8")) == {
+        "success": False,
+        "message": mock_file_info,
+    }
+
+
+def test_bulk_extractor_if_missing_reports_exist(
+    admin_client, dashboard_uuid, file, mocker
+):
+    mocker.patch("elasticSearchFunctions.get_client")
+    mocker.patch(
+        "elasticSearchFunctions.get_transfer_file_info",
+        return_value={
+            "filename": "piiTestDataCreditCardNumbers.txt",
+            "relative_path": "test-bulk-extract-a239e3e1-0391-46da-94d7-25a3e8509b45/data/objects/piiTestDataCreditCardNumbers.txt",
+            "fileuuid": "259e911f-c33d-4e11-99c5-7cfc2abe7fb0",
+            "sipuuid": "a239e3e1-0391-46da-94d7-25a3e8509b45",
+            "accessionid": "",
+            "status": "backlog",
+            "created": 1713465340.0109284,
+            "modification_date": "2023-09-28",
+            "size": 0.00026416778564453125,
+            "tags": [],
+            "file_extension": "txt",
+            "bulk_extractor_reports": ["ccn"],
+            "format": [
+                {"puid": "x-fmt/111", "format": "Generic TXT", "group": "Text (Plain)"}
+            ],
+            "pending_deletion": False,
+        },
+    )
+
+    response = admin_client.get(reverse("file:bulk_extractor", args=[file.uuid]))
+    assert response.status_code == 400
+
+    assert response.json() == {
+        "success": False,
+        "message": "Requested file is missing the following requested reports: pii",
+    }
+
+
+@mock.patch("components.file.views.len", side_effect=[0])
+def test_bulk_extractor_validates_reports_parameter_with_len_mock(
```

Comment:
There's no need to mention the mechanics of the test in its name (suppose we need to change how the test works, we'd need to update the name). Names should reflect the purpose of the test not how it does it.